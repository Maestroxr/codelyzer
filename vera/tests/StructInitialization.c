
/*OK*/
struct first
{
	int something;
}

/*********/

int main() 
{
	return 0;
	
}


