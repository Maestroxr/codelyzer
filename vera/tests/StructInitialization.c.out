b"Directory /task/student/sanitizer doesn't exist.\r\n"